components/lists/Friends.tsx

```diff
@@ -6,14 +6,17 @@ import api from "@/service"
 import { User } from "@/types"
 import { useStatus } from "@/contexts/StatusContext"
 import { useSignal } from "@/hooks/useSignal"
+import { FriendsSkeleton } from "@/components/cards/FriendsSkeleton"
 
 export default function FriendsList() {
   const { friends, setFriends } = useStatus()
   const [friendsList, setFriendsList] = useState<User[]>([])
   const { signal } = useSignal()
+  const [isLoadingFriendsList, setIsLoadingFriendsList] = useState(false)
```

Comment:
isn't it better to have the initial value as `true`
```suggestion
  const [isLoadingFriendsList, setIsLoadingFriendsList] = useState(true)
```